server/src/main/java/org/elasticsearch/action/admin/indices/stats/IndicesStatsResponse.java

```diff
@@ -208,11 +213,13 @@ protected Iterator<ToXContent> customXContentChunks(ToXContent.Params params) {
                                 builder.field("status", indexStats.getState().toString().toLowerCase(Locale.ROOT));
                             }
                             builder.startObject("primaries");
-                            indexStats.getPrimaries().toXContent(builder, p);
+
+                            var pp = new ToXContent.DelegatingMapParams(Map.of(DenseVectorStats.INCLUDE_PER_FIELD_STATS, "true"), params);
+                            indexStats.getPrimaries().toXContent(builder, pp);
```

Comment:
Does this mean we always default to include per field stats? Or does something upstream set it to false? I am confused by the delegate pattern.